src/diagonal.works/b6/cmd/b6-ingest-gtfs/b6-ingest-gtfs.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"runtime"
+
+	"diagonal.works/b6/ingest/compact"
+	"diagonal.works/b6/ingest/gtfs"
+
+	_ "github.com/apache/beam/sdks/go/pkg/beam/io/filesystem/local"
+)
+
+// TODO: add monitoring
```

Comment:
we can do that in a separate cl + add monitoring for all binaries ?

Reply:
Yeah, that seems fine.